File: Perl/fibonacci.pl
use v5.10;

sub fibonacci
{
    my $x = shift(@_);
    
    if ($x == 1 or $x == 2)
    {
        return(1);
    }
    else
    {
        return(fibonacci($x-1) + fibonacci($x-2));
    }
}

for my $x (1..9)
{
    say(fibonacci($x));
}

File: Ruby/fibonacci.rb
def fibonacci(n)
  if n === 1 || n === 2
    return 1
  end

  return fibonacci(n - 1) + fibonacci(n - 2)
end

for i in 1..10 do
  res = fibonacci(i)
  puts ("fibonacci of #{i} = #{res}\n")
end
